### PR TITLE
tinyaes and folder restructuring

### PR DESCRIPTION
**Pull request by manumanuk**

### CODE REVIEW

File: TinyAESEncryption/example.c [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    printf("CIPHERTEXT: %s\n", msg);
+
+    printf("\n\nDecrypting...\n");
+
+    // Initialize context
+    struct AES_ctx decCtx;
+    AES_init_ctx_iv(&decCtx, key, iv);
+
+    // Decrypt
+    AES_CTR_xcrypt_buffer(&decCtx, msg, 48);
```
Comment (christopherlam888) [inline TinyAESEncryption/example.c:30]: Recommend adding comment indicating 48 is the length of the encrypted string.

**Review by christopherlam888 — CHANGES_REQUESTED** [review]
Looks good.

**Review by christopherlam888 — APPROVED** [review]
Looks good.

**Review by Navtajh04 — APPROVED** [review]
LGTM

**Pull request merged**